SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -29,6 +30,63 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKEditor);
 
 #define LOCTEXT_NAMESPACE "FSpatialGDKEditor"
 
+namespace 
+{
+
+bool CheckAutomationToolsUpToDate()
+{
+#if PLATFORM_WINDOWS
+	FString RunUATScriptName = TEXT("RunUAT.bat");
+	FString CmdExe = TEXT("cmd.exe");
+#elif PLATFORM_LINUX
+	FString RunUATScriptName = TEXT("RunUAT.sh");
+	FString CmdExe = TEXT("/bin/bash");
+#else
+	FString RunUATScriptName = TEXT("RunUAT.command");
+	FString CmdExe = TEXT("/bin/sh");
+#endif
+
+	FString UatPath = FPaths::ConvertRelativePathToFull(FPaths::EngineDir() / TEXT("Build/BatchFiles") / RunUATScriptName);
+	
+	if (!FPaths::FileExists(UatPath))
+	{
+		FFormatNamedArguments Arguments;
+		Arguments.Add(TEXT("File"), FText::FromString(UatPath));
+		FMessageDialog::Open(EAppMsgType::Ok, FText::Format(LOCTEXT("RequiredFileNotFoundMessage", "A required file could not be found:\n{File}"), Arguments));
+
+		return false;
+	}
+
+#if PLATFORM_WINDOWS
+	FString FullCommandLine = FString::Printf(TEXT("/c \"\"%s\" -list\""), *UatPath);
```

Comment:
Do we want `/c` here? That's creating a new window no?

Reply:
That what the UATHelper uses. Docs says execute command then terminates.